no_log.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- hosts: all
+  gather_facts: false
+  tasks:
+    - name: task with stdout
+      shell: echo {{ secret_value }}
+      no_log: True
+
+    - name: task with stderr
+      shell: cat {{ secret_value }}
+      no_log: True
+      ignore_errors: yes
```

Comment:
@simfarm neither do I, but *why* was it failing?  You shouldn't need to `ignore_errors` here.